internal/aez/htr.go

```diff
@@ -144,12 +146,12 @@ func (htr *HTR) Report() HTRReport {
 }
 
 //CSVHeaders returns the field names
-func (htr HTR) CSVHeaders() []string {
+func (htr *HTR) CSVHeaders() []string {
```

Comment:
What about the methods on lines 32--46? (Github won't let me comment up there.)

Reply:
No really worth it since the type `htr` it's just a uint16... it should probably have another name as to not confuse it with the struct.